pkg/common/types.go

```diff
@@ -30,6 +30,12 @@ const (
 
 	GCPManagedClusterKind         = "GCPManagedCluster"
 	GCPManagedClusterKindProvider = "gke"
+
+	MonitoringAgentPrometheus = "prometheus-agent"
```

Comment:
Maybe they should bé in thé monitoring config and not in common?

Reply:
I moved this into a new common/monitoring package, I keep this into a separate package to avoid later circular import problem with the upcoming implementation.